pkg/cmd/kafka/create/create.go

```diff
@@ -107,22 +107,22 @@ func NewCreateCommand(f *factory.Factory) *cobra.Command {
 				return flagutil.InvalidValueError("output", opts.outputFormat, validOutputFormats...)
 			}
 
-			validMarketplaceAcctIDs, err := accountmgmtutil.GetValidMarketplaceAcctIDs(f.Context, f.Connection)
+			validMarketplaces, err := accountmgmtutil.GetValidMarketplaces(f.Context, f.Connection)
 			if err != nil {
 				return err
 			}
 
-			if opts.marketplaceAcctId != "" && !flagutil.IsValidInput(opts.marketplaceAcctId, validMarketplaceAcctIDs...) {
-				return flagutil.InvalidValueError(FlagMarketPlaceAcctID, opts.marketplaceAcctId, validMarketplaceAcctIDs...)
+			if opts.marketplace != "" && !flagutil.IsValidInput(opts.marketplace, validMarketplaces...) {
+				return flagutil.InvalidValueError(FlagMarketPlace, opts.marketplace, validMarketplaces...)
 			}
 
-			validMarketplaces, err := accountmgmtutil.GetValidMarketplaces(f.Context, f.Connection)
+			validMarketplaceAcctIDs, err := accountmgmtutil.GetValidMarketplaceAcctIDs(f.Context, f.Connection, opts.marketplace)
```

Comment:
I just realized. Why we do separate call to AMS ( very expensive)
This will break kas-installer that doesn't have ams also it is not performant.
It will be better to extend exisiting quota object we have

WDYT?

Basically if we do --skip-checks we should not make any call to ams

Reply:
> It will be better to extend exisiting quota object we have

That will be better, will do it